### PR TITLE
fix(l10n): fix translations for init scripts

### LINKED ISSUE #41549: [Bug] [NC 28b3]: Translation "New folder"
### ⚠️ This issue respects the following points: ⚠️

- [X] This is a **bug**, not a question or a configuration/webserver/proxy issue.
- [X] This issue is **not** already reported on [Github](https://github.com/nextcloud/server/issues?q=is%3Aopen+is%3Aissue+label%3Abug) OR [Nextcloud Community Forum](https://help.nextcloud.com/) _(I've searched it)_.
- [X] Nextcloud Server **is** up to date. See [Maintenance and Release Schedule](https://github.com/nextcloud/server/wiki/Maintenance-and-Release-Schedule) for supported versions.
- [X] I agree to follow Nextcloud's [Code of Conduct](https://nextcloud.com/contribute/code-of-conduct/).

### Bug description

The translation "New folder" is not displayed in the menu for adding a file or folder for the "Files" application.

obraz

The translation is in files l10n.

### Steps to reproduce

1. Run the files application.
2. Click on "+" to create a new folder.

### Expected behavior

Translated text "New folder".

### Installation method

None

### Nextcloud Server version

28

### Operating system

None

### PHP engine version

None

### Web server

None

### Database engine version

None

### Is this bug present after an update or on a fresh install?

None

### Are you using the Nextcloud Server Encryption module?

None

### What user-backends are you using?

- [ ] Default user-backend _(database)_
- [ ] LDAP/ Active Directory
- [ ] SSO - SAML
- [ ] Other

### Configuration report

_No response_

### List of activated Apps

_No response_

### Nextcloud Signing status

_No response_

### Nextcloud Logs

_No response_

### Additional info

_No response_

### PR DESCRIPTION
**Pull request by skjnldsv**
Fix https://github.com/nextcloud/server/issues/41549

### CODE REVIEW

**Review by juliusknorr — COMMENTED** [review]
Psalm needs some love but otherwise seems good 👍

File: lib/public/Util.php [hunk 1]
```diff
@@ -235,7 +243,7 @@ public static function getScripts(): array {
 	 * @param string $languageCode language code, defaults to the current locale
 	 * @since 8.0.0
 	 */
-	public static function addTranslations($application, $languageCode = null) {
+	public static function addTranslations($application, $languageCode = null, $init = false) {
```
[the remark this answers is not in the document]
Comment (skjnldsv) [reply lib/public/Util.php:246]: We could add all those in init, but I'm afraid perfs could suffer :)
Especially bandwith and requests usage :thinking:

**Review by AndyScherzinger — APPROVED** [review]

**Review by blizzz — DISMISSED** [review]
breaks api

File: lib/public/Util.php [hunk 2]
```diff
@@ -235,7 +241,7 @@ public static function getScripts(): array {
...
```
Comment (blizzz) [inline lib/public/Util.php:244]: please reflect added parameter in the doc block. Consider adding a type hint.

Comment (skjnldsv) [reply lib/public/Util.php:245]: Done

**Review by blizzz — APPROVED** [review]

**Pull request merged**